xapian-applications/omega/configure.ac

```diff
@@ -261,6 +261,17 @@ PKG_CHECK_MODULES([POPPLER], [poppler-cpp], [
                      OMINDEX_MODULES="$OMINDEX_MODULES omindex_pdf"])
      LIBS=$save_LIBS],[ ])
 
+dnl check if libe-book is available.
+PKG_CHECK_MODULES([LIBEBOOK], [libe-book-0.1,librevenge-0.0,librevenge-generators-0.0,librevenge-stream-0.0], [
+     save_LIBS=$LIBS
+     LIBS="$LIBS $LIBEBOOK_CFLAGS $LIBEBOOK_LIBS"
+     AC_CHECK_HEADERS([librevenge-0.0/librevenge-generators/librevenge-generators.h \
+                       librevenge-0.0/librevenge-stream/librevenge-stream.h \
+                       libe-book/libe-book.h],
+                       [AC_DEFINE(HAVE_LIBEBOOK, 1, [Define HAVE_LIBEBOOK if the library is available])
+                        OMINDEX_MODULES="$OMINDEX_MODULES omindex_ebook"])
```

Comment:
I don't think we need this `AC_CHECK_HEADERS` check if we're using pkg-config to report the flags to use.

For poppler we needed an additional test to catch the libstdc++ vs libc++ compatibility issue, because poppler provides a C++ API.

Reply:
I didn't realize that, sorry. I am going to fix it.

Reply:
> because poppler provides a C++ API

Actually, both libe-book and librevenge are C++ APIs too.  However, there's a probably important difference - they don't use anything from the C++ standard library from their public API headers.  That means they may well avoid the problems we had with poppler.